internal/service/cloudbackupsnapshotexportjob/data_source_cloud_backup_snapshot_export_job.go

```diff
@@ -56,8 +56,9 @@ func DataSource() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"export_id": {
-							Type:     schema.TypeString,
-							Computed: true,
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: fmt.Sprintf(constant.DeprecationParamByVersion, "1.18.0"),
```

Comment:
would adjust docs if needed and also add a changelog entry mentioning the deprecation ([example](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/contributing/changelog-process.md#deprecations))

Reply:
Thanks. Added in 9187979